mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactLoggedModelView.js

```diff
@@ -93,6 +98,15 @@ class ShowArtifactLoggedModelView extends Component {
     );
   }
 
+  loadModelCodeText(modelPath, flavor) {
+    return (
+      `import mlflow\n` +
+      `logged_model = '${modelPath}'\n\n` +
+      `# Load model.\n` +
+      `loaded_model = mlflow.${flavor}.load_model(logged_model)\n`
+    );
+  }
```

Comment:
@harupy This is perfect for everything except MLeap. MLeap models can't be reloaded in Python. Perhaps we can add some Java example for MLeap?

Reply:
Got it. If so, I'd prefer to show nothing for now.

Reply:
Sounds good!